icu4c/source/data/brkitr/ja.txt

```diff
@@ -12,14 +12,64 @@ ja{
         line_strict:process(dependency){"line_cj.brk"}
         line_strict_phrase:process(dependency){"line_phrase_cj.brk"}
     }
-    particles{
-         か, かしら, から, が, くらい, けれども, こそ,
-         さ, さえ, しか, だけ, だに, だの, て, で, でも,
-         と, ところが, とも, な, など, なり, に, ね, の,
-         ので, のに, は, ば, ばかり, へ, ほど, まで, も,
-         や, やら, よ, より, わ, を
-    }
-    auxVerbs {
-        です, でしょ, でし, ます, ませ, まし
+    extensions{
+        // Extensions.
```

Comment:
Thanks Hiro!
Hopefully I finally get it... These strings are needed for grammatical reasons (particles etc.), but cjdict also needs to include them because otherwise these particles/verbs etc. would be split into single characters, right?
If so, then I am not sure that we need to tweak this comment further. @pedberg-icu are you satisfied with this comment?

Reply:
@markusicu Yes, I am satisfied with that comment.